manifests/init.pp

```diff
@@ -0,0 +1,176 @@
+  Optional[Array[String[1]]]  $scriptwhitelist                   = undef,
+  Optional[Array[String[1]]]  $shared_lib_whitelist              = undef,
+  Integer[0,1]                $show_lock_msgs                    = 1,
+  Integer[0,3]                $show_summary_time                 = 3,
+  Integer[0,1]                $show_summary_warning_number       = 0,
+  Integer[0,1]                $skip_inode_check                  = 0,
+  String[1]                   $ssh_config_dir                    = '/etc/ssh',
+  Optional[Array[String[1]]]  $startup_paths                     = undef, #suggestion: /etc/rc.d /etc/rc.local ??
+  Optional[String[1]]         $stat_cmd                          = undef,
+  Optional[Array[String[1]]]  $suspscan_dirs                     = undef,
+  Integer[0]                  $suspscan_maxsize                  = 1024000,
+  String[1]                   $suspscan_temp                     = '/dev/shm',
+  Integer[0]                  $suspscan_thresh                   = 200,
+  Optional[Array[String[1]]]  $suspscan_whitelist                = undef,
+  Optional[Array[String[1]]]  $syslog_config_file                = undef,
+  String[1]                   $tmpdir                            = '/var/lib/rkhunter',
+  Optional[Array[String[1]]]  $uid0_accounts                     = undef,
+  Array[String[1]]            $unhide_tests                      = ['sys'],
+  Array[String]             $unhidetcp_opts                    = [''],
+  Optional[Array[String[1]]]  $update_lang                       = undef,
+  Integer[0,1]                $update_mirrors                    = 1,
+  Integer[0,1]                $updt_on_os_change                 = 0,
+  Integer[0,1]                $use_locking                       = 1,
+  Optional[Array[String[1]]]  $user_fileprop_files_dirs          = undef,
+  Integer[0,1]                $use_sunsum                        = 0,
+  String[1]                   $use_syslog                        = 'NONE', #recommend authpriv.notice??
+  Integer[0,1]                $warn_on_os_change                 = 1,
+  Optional[String[1]]         $web_cmd                           = undef, #shuold keep? rkhunter auto selects one
+  Integer[0,1]                $whitelisted_is_white              = 0,
+  Optional[Array[String[1]]]  $writewhitelist                    = undef,
+  Optional[Array[String[1]]]  $xinetd_allowed_svc                = undef,
+  Optional[String[1]]         $xinetd_conf_path                  = undef
+) {
+
+  # Some tests require single-purpose tools, if rkhunter has
+  # them then it will use them. Unhide is one such tool. 
+  package { 'unhide':
+    ensure => $package_ensure
+  }
+
+  package { 'rkhunter':
+    ensure  => $package_ensure,
+    require => Package['unhide']
+  }
+
+  file { $rkhunter_conf_file:
+    ensure  => 'file',
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0640',
+    path    => '/etc/rkhunter.conf',
```

Comment:
The path should not be set or should be set to $rkhunter_conf_file.    This will change the name of the resource but not the location of the file.